### DISCUSSION TITLE
SDFat Audio Source

### DISCUSSION THREAD

**Discussion by jonascant** [pschatzmann/arduino-audio-tools#1646] [Q&A] [answered]
I am using the SDFat arduino library to do actions on a file, for example transmit it to another esp32 via network.
I would also like to play wav files using SDFat. 
My question is, do i need to call `SdSpiConfig sdcfg(PIN_AUDIO_KIT_SD_CARD_CS, DEDICATED_SPI, SD_SCK_MHZ(10) , &SPI);`, since i already initialized the sd card using `sd.begin(sdChipSelectPin,SPI_SPEED)`

**Comment by pschatzmann** [comment, marked as answer]
I am missing some context: What functionality are you planning to use ?
If you check the SDFat documentation you will see that you can alternative.y call sd.begin(sdcfg) 

But in any case this has nothing to do with my Audio Library!

**Reply by jonascant** [reply]
Audio files will be recorded to SDFat using your library. I assume your library will initialize the sd card. later, i want to transfer the files (that part is not related to your library), so i need to initialize the SDFat again. I am worried about problems related to that.

**Comment by pschatzmann** [comment]
There is no specific functionality in my library for recording files: It's just relying on the Arduino Stream API which are provided by the different SD implementations!
